### ISSUE TITLE
Error in angr example

### ISSUE THREAD

**Issue by burtonwilliamt** [Vector35/binaryninja-api#729]
I was trying out the example that you have for angr, and it seems a refactor a while ago didn't quite work. [This line](https://github.com/Vector35/binaryninja-api/blob/master/python/examples/angr_plugin.py#L140) shouldn't say 
```python
show_message_box("Angr Solve", "You have not specified a goal instruction.\n\n" +
	"Please right click on the goal instruction and select \"Find Path to This Instruction\" to " +
	"continue.", MessageBoxButtonSet.OKButtonSet, MessageBoxButtonSet.ErrorIcon)
```
but should say
```python
show_message_box("Angr Solve", "You have not specified a goal instruction.\n\n" +
	"Please right click on the goal instruction and select \"Find Path to This Instruction\" to " +
	"continue.", MessageBoxButtonSet.OKButtonSet, MessageBoxIcon.ErrorIcon)
```
 specifically `MessageBoxButtonSet.ErrorIcon` -> `MessageBoxIcon.ErrorIcon`
Unless there is actually an error icon button but it isn't being imported, or perhaps I'm doing something wrong?

**Comment by joshwatson** [comment]
Nope, that's definitely a typo. If you want, you could file a PR to fix it 😃

**Comment by burtonwilliamt** [comment]
Made #730 
Also, it's commit 500, woohoo!

**Comment by psifertex** [comment]
Fixed in builds > 1.0.811
